lib/3scale_toolbox/commands/activedocs_command/list_command.rb

```diff
@@ -42,8 +43,26 @@ def print_header(fields_to_show)
             puts fields_to_show.map { |e| e.upcase }.join("\t")
           end
 
+          def service_id_filter
+            options[:'service-id']
+          end
+
+          def filters
```

Comment:
Making extensible with a list of filters sounds good, but I think it is over engineering as I do not expect to have more filters to this command. Maybe yes? who knows.

However over engineering is nice sometimes and smart code is nice even if it is never extended. So I propose to implement an extensible filter list. There are different ways to do it. What I would do is to create a list of functions with the same interface, enumerable element as input param and filtered enumerable element as ouput. Each function would then be applied in a serial pipeline (one after the other connecting the output of the previous one with the input of the next one)